File: stats/KS.R
c0<-read.table("C:/Users/Rui/Desktop/nc0.csv",sep=",",header=TRUE)
c0$Freq=c0$Freq/100
c0vec=rep(c0$Val,c0$Freq)
c2<-read.table("C:/Users/Rui/Desktop/nc2.csv",sep=",",header=TRUE)
c2$Freq=c2$Freq/100
c2vec=rep(c2$Val,c2$Freq)
cresult=ks.test(c0vec,c2vec,exact=TRUE)
cresult$p.value

File: stats/KS.py
import pandas
import numpy
from scipy import stats
c0=pandas.read_csv("C:/Users/Rui/Desktop/nc0.csv")
V=c0['Val'].as_matrix()
c0F=c0['Freq'].tolist()
c0F=[int(x/250) for x in c0F]
c0=numpy.repeat(V,c0F,axis=0)
c2=pandas.read_csv("C:/Users/Rui/Desktop/nc2.csv")
c2F=c2['Freq'].tolist()
c2F=[int(x/250) for x in c2F]
c2=numpy.repeat(V,c2F,axis=0)
print(stats.ks_2samp(c0,c2))
#1000 for coding
